modules/reference/pages/feature/feature-overview.adoc

```diff
@@ -48,6 +48,24 @@ The runtime contains default configuration settings so that the configuration yo
 Specify the features that you need, along with any additions or overrides to the default settings, in the `server.xml` file.
 For details about the server configuration, see the xref:config/server-configuration-overview.adoc[server configuration overview].
 
+You can enable a whole set of features all at once using the convenience features. Use these convenience features in the feature manager of your server configuration to add support for your Open Liberty server.
+The following are considered as convenience features for Jakarta 9.1:
```

Comment:
suggested edit- use the definition list syntax:

The following convenience features are available for the for Jakarta EE platform:
feature:jakartaee[display=Jakarta EE Platform]::
This feature enables all features that are associated with the Jakarta EE platform

feature:webProfile[display=Jakarta EE Web Profile]::
This feature enables only features that are associated with the lightweight link:https://jakarta.ee/specifications/webprofile/9/[Jakarta EE Web Profile].

feature: jakartaeeClient[display=Jakarta EE Application Client]::
This feature enables all features that are associated with Jakarta EE link:https://jakarta.ee/specifications/platform/9/jakarta-platform-spec-9.html#a3294[Application Client] functions.

Additionally, the feature:enterpriseBeans[Jakarta Enterprise Beans] convenience feature automatically enables all Open Liberty features that support enterprise beans functions.